pwa/src/apiService/resources/object.ts

```diff
@@ -24,10 +24,14 @@ export default class Sources {
     return data;
   };
 
-  public getAllFromEntity = async (entityId: string): Promise<any> => {
-    const { data } = await Send(this._instance, "GET", `/admin/objects?_self.schema.id=${entityId}&limit=200`);
+  public getAllFromEntity = async (entityId: string, page: number): Promise<any> => {
```

Comment:
In other implementations, we call it `currentPage` instead of `page`. I'd opt to make it consistent.